docs/tutorials/build-a-portfolio-website-with-react-webiny-apollo.md

```diff
@@ -0,0 +1,1048 @@
+
+Press the `SAVE` button on the right corner. Go back and create `Projects` content model.
+
+These are the fields we will use for our `Projects`:
+- `Title` with type `TEXT`
+- `Cover` with type `FILES`
+- `Content` with type `LONG TEXT`
+- `Link` with type `TEXT`
+- `CTA` with type `TEXT`
+
+![Projects content model](/img/tutorials/build-a-portfolio-webste-with-react-webiny-apollo/projects-content-model.png)
+
+### 1.6 Filling the data for the content models
+
+Here you will create your creative content for Blogs, by filling all the input fields that we created in the above step.
+
+![Blogs data](/img/tutorials/build-a-portfolio-webste-with-react-webiny-apollo/blogs-data.png)
+
+You will follow the same process to create content for your project’s content models.
+
+### 1.7 Access Token
+
+Now that we have our data created, we can go ahead and check the `API information` of our `backend` project. We have all of the URLs we need to continue building our project. The one we need for now is the Headless CMS - Content Delivery API `cms/read/production`, where Webiny exposes a GraphQL Playground you can use to easily test and debug your API calls.
+
+![API Information](/img/tutorials/build-a-portfolio-webste-with-react-webiny-apollo/api-information.png)
+
+Open the Content Delivery API which ends with the `/cms/read/production`. Now, let’s write some queries to read our Blog content.
+``` 
+query getBlogs {
+  listBlogs {
+    data {
+      title
+      subtitle
+      link
+    }
+  }
+} 
+```
+
+![Get blogs query](/img/tutorials/build-a-portfolio-webste-with-react-webiny-apollo/get-blogs-query.png)
+
+To be able to read our data, you need to create an `Access Token` in your backend project. Go ahead at the `Settings` menu and create an access token. Then copy and paste it to the HTTP HEADERS for the **Authorization** in the GraphQL Playground. Check out the image below.
+
+![Access token](/img/tutorials/build-a-portfolio-webste-with-react-webiny-apollo/access-token.png)
+
+Now that we are getting our data, it’s time to start building the `Front-end`.
+
+### 2. Front-end
+
+We will create a `create-react-project` front-end application. On top of that, we will use the Material-UI framework. The React component library, based on Google Material Design, allows us a fast and easy usage of stylized web components. With basic knowledge of the React framework, you can build a useful looking material application.
+
+First, let's start by creating a react application from scratch by running this command:
+
+```tsx
+npx create-react-app my-app --template typescript
+cd frontend
+npm run start
+```
+One of the advantages of using TypeScript is that Type errors will show up in the same console as the building one. You'll have to fix these type errors before you continue development or build your project. For advanced configuration, [see here](https://create-react-app.dev/docs/advanced-configuration/).
```

Comment:
We can rephrase this sentence a little to make it shorter, what do you think?

` You'll have to fix these type errors to continue development or build the project.`